src/CLI/clitree/cli-xml/interface.xml

```diff
@@ -159,5 +191,87 @@ limitations under the License.
             <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_mtu ${iface} 9100 </ACTION>
         </COMMAND>
     </VIEW>
+    <VIEW
+        name="configure-if-mgmt-view"
+        prompt="${SYSTEM_NAME}(conf-if-${iface})# "
+        depth="2"
+        >
+        <!-- Inheritance -->
+        <NAMESPACE
+            ref="configure-view"
+            help="false"
+            completion="false"
+            />
+        <COMMAND
+            name="no"
+            help="Negate a command or set its defaults"
+            />
+        <COMMAND
+            name="shutdown"
+            help="Disable the interface">
+            <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_enabled ${iface} False </ACTION>
+        </COMMAND>
+        <COMMAND
+            name="no shutdown"
+            help="Enable the interface">
+            <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_enabled ${iface} True </ACTION>
+        </COMMAND>
+        <COMMAND
+            name="description"
+            help="Textual description">
+	    <PARAM
+                name="desc"
+                help="Textual description of the interface"
+                ptype="STRING" />
+            <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_description ${iface} ${desc} </ACTION>
+        </COMMAND>
+        <COMMAND
+            name="no description"
+            help="Remove description" >
+            <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_description ${iface} "" </ACTION>
+        </COMMAND>
+        <COMMAND
+            name="mtu"
+            help="Configure MTU">
+	    <PARAM
+                name="mtu" 
+                help="MTU of the interface"
+                ptype="RANGE_MTU" />
+            <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_mtu ${iface} ${mtu} </ACTION>
+        </COMMAND>
+	<COMMAND
+            name="no mtu"
+            help="Remove MTU">
+            <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_mtu ${iface} 9100 </ACTION>
+        </COMMAND>
+        <COMMAND
+            name="autoneg"
+            help="Configure autoneg">
+	    <PARAM 
+                name="autoneg" 
```

Comment:
Do we have autoneg support?

Reply:
For management interface will be adding the support